browser/src/control/Control.NotebookbarCalc.js

```diff
@@ -1832,6 +1832,36 @@ L.Control.NotebookbarCalc = L.Control.NotebookbarWriter.extend({
 				],
 				'vertical': 'true'
 			},
+			{
+				'type': 'container',
+				'children': [
+					{
+						'type': 'toolbox',
+						'children': [
+							{
+								'id': 'data-add-name',
+								'type': 'toolitem',
+								'text': _UNO('.uno:AddName', 'spreadsheet'),
+								'command': '.uno:AddName',
+								'accessibility': { focusBack: true,	combination: 'DD', de: null }
```

Comment:
@gokaysatir do we need to always put `de: null` ?

Reply:
@eszkadev no we shouldn't need that. I had put them as a placeholder in the first implementation. Because many of them were subject to change.